deploy/modules/aws/tidb-cluster/cluster.tf

```diff
@@ -0,0 +1,14 @@
+module "tidb-cluster" {
+  source = "../../share/tidb-cluster-release"
+
+  cluster_name               = var.cluster_name
+  cluster_version            = var.cluster_version
+  pd_count                   = var.pd_count
+  tikv_count                 = var.tikv_count
+  tidb_count                 = var.tidb_count
+  tidb_cluster_chart_version = var.tidb_cluster_chart_version
+  override_values            = var.override_values
+  local_exec_interpreter     = var.local_exec_interpreter
```

Comment:
Can this work if a user does not provide `local_exec_interpreter`?

Reply:
Yes, there is a reasonable default `["bash", "-c"]`